content/en/docs/Installation/installation-guide/accessing-kiali.md

```diff
@@ -42,56 +42,84 @@ have the necessary privileges to perform port forwarding.
+defined, allowing you to access
+the Kiali UI through the Ingress. By default, an Ingress will not be created. You can
+enable a simple Ingress by setting `spec.deployment.ingress.enabled` to `true` in the Kiali
+CR (a similar setting for the server Helm chart is available if you elect to install Kiali
+via Helm as opposed to the Kiali Operator).
+
+Exposing Kiali externally through this `spec.deployment.ingress` mechanism is a
+convenient way of exposing Kiali externally but it will not necessarily work or
+be the best way to do it because the way in which you should expose Kiali
+externally will be highly dependent on your specific cluster environment and
+how services are exposed generally for that environment.
+
+{{% alert color="info" %}}
+When installing on an OpenShift cluster, an OpenShift Route will be installed (not an Ingress).
+This Route *will* be installed by default unless you explicitly
+disable it via `spec.deployment.ingress.enabled: false`. Note that the Route is required
+if you configure Kiali to use the auth strategy of `openshift` (which is the default
+auth strategy Kiali will use when installed on OpenShift).
+{{% /alert %}}
+
+The default Ingress that is created will be configured for a typical NGinx implementation. If you have your own
+Ingress implementation you want to use, you can override the default configuration through
+the settings `spec.deployment.ingress.override_yaml` and `spec.deployment.ingress.class_name`.
+More details on customizing the Ingress can be found below.
 
-To find your Ingress IP or domain name, as per
+The Ingress IP or domain name should then be used to access the Kiali UI. To find your Ingress IP or domain name, as per
 [the Kubernetes documentation](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/#create-an-ingress),
-try the following command (doesn't work if using Minikube):
+try the following command (though this may not work if using Minikube without the ingress addon):
 
 ```
 kubectl get ingress kiali -n istio-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
 ```
 
 If it doesn't work, unfortunately, it depends on how and where you had setup
 your cluster. There are several Ingress controllers available and some cloud
-providers have their own controller or preferred exposure method. Please, check
+providers have their own controller or preferred exposure method. Check
 the documentation of your cloud provider. You may need to customize the
-pre-installed Ingress rule, or to expose Kiali using a different method.
+pre-installed Ingress rule or expose Kiali using a different method.
 
 ### Customizing the Ingress resource
 
-By default, a catch-all Ingress resource is created to route traffic to Kiali.
-Most likely, you will need a more specific Ingress resource that routes traffic
+The created catch-all Ingress resource will route traffic to Kiali. You may need a more specific Ingress resource that routes traffic
```

Comment:
I'm not sure I like the rewording of the first sentence: _The created catch-all Ingress resource will route traffic to Kiali._.

It reads a little confusing.

Reply:
suggestions welcome? I actually don't know what "catch-all" meant in the original...I thought about removing that term.

Reply:
A "catch-all" ingress is the one that will always route traffic to the specified service, no matter what you type in the URL.

I think this is not really a true "catch-all" because a path must be matched (by default, the `/kiali` path). However, it will reply in any domain.

Reply:
how about this: https://github.com/kiali/kiali.io/pull/466/commits/e9a8325f5476a8d199e1b47caf3430ff3a1ea438

Reply:
Ahhh! Much better! It's more precise than the currently published wording :)